00c-Addressing-Control-Failures/Readme.md

```diff
@@ -189,14 +189,16 @@ the permissions required section below.
 [Back to top...](Readme.md#contents)
 ### Starting attestation
       
-The AzSK subscription and services scan cmdlets now support a new switch called *AttestControls*. When this switch is specified, 
+The AzSK subscription and services scan cmdlets now support a new switch called *ControlsToAttest*. When this switch is specified, 
 AzSK enters 'attest' mode immediately after a scan is completed. This ensures that attestation is done on the basis of the most current
 control statuses.
 
 All controls that have a technical evaluation status of anything other than 'Passed' (i.e., 'Verify' or 'Failed' or 'Manual' or 'Error') are considered 
 valid targets for attestation.
 
-To manage attestation flow effectively, 4 options are provided for the *AttestControls* switch to specify which subset of controls to target for attestation. These are described below:
+> **Note**: AzSK does not support attestation of some selected controls, and therefore fixing them is mandatory.
```

Comment:
I think we should rather say-
'Some controls are very crucial from security stand point and hence AzSK does not support attesting them.'